azext_edge/edge/command_map.py

```diff
@@ -172,3 +173,15 @@ def load_iotops_commands(self, _):
         cmd_group.show_command("show", "show_registry")
         cmd_group.command("list", "list_registries")
         cmd_group.command("delete", "delete_registry")
+
+    with self.command_group(
```

Comment:
assuming this is a temp command - put a TODO somewhere that says this is a temp command (maybe here in command map?)

We would want to mark it correctly when GAing to minimize breaking changes

Reply:
why is this temp command?